### PR TITLE
Documentation - Brief description of all open source projects available 

### PR DESCRIPTION
**Pull request by atharvapurdue**
I have added a brief description of all available open source models for speech translation.
I have also added which models we will be considering for our project. 
The initial list of 9 models is short-selected to 3 - 
1. Whisper
2. SpeechBrain
3. Kadli

For on-mobile over-the-air translation -  DeepSpeech

### CODE REVIEW

**Review by yash-1708 — APPROVED** [review]

**Review by marktrovinger — APPROVED** [review]
Looks good, approved!

**Comment by atharvapurdue** [comment]
@winniehar can you review the PR?

**Review by atharvapurdue — COMMENTED** [review]
All the required changes are done.
Thanks for pointing out the grammar mistakes @winniehar !
@zeshengchen could we give a honorary 0.1 point to Winnie?

File: README.md [hunk 1]
```diff
@@ -12,4 +12,66 @@ Another model we are investigating is [ESPNet](https://github.com/espnet/espnet)
...
+
+We are currently exploring Kaldi. A good starting point is this [tutorial](https://www.assemblyai.com/blog/
+kaldi-speech-recognition-for-beginners-a-simple-tutorial/)
+Brief Description -
+[Kaldi](https://github.com/kaldi-asr/kaldi) is a speech recognition toolkit that has been widely popular in the research community for many years.
+Like DeepSpeech, Kaldi has good out-of-the-box accuracy and supports the ability to train your own models. It’s also been thoroughly tested--a lot of companies currently use Kaldi in production and have used it for a while--making more developers confident in its application.
+
+# Wav2Letter
+
+We will not be going with Wav2Letter as it is majorly writtten in C++
```
[the remark this answers is not in the document]
Comment (winniehar) [reply README.md]: @atharvapurdue spelling for written, 2 t's only

**Review by winniehar — APPROVED** [review]

**Pull request merged**